js/client-rooms.js

```diff
@@ -118,14 +118,16 @@
 		title: 'Battles',
 		isSideRoom: true,
 		events: {
-			'change select[name=elofilter]': 'refresh'
+			'change select[name=elofilter]': 'refresh',
+			'click .search': 'refresh'
 		},
 		initialize: function () {
 			this.$el.addClass('ps-room-light').addClass('scrollable');
 			var buf = '<div class="pad"><button class="button" style="float:right;font-size:10pt;margin-top:3px" name="close"><i class="fa fa-times"></i> Close</button><div class="roomlist"><p><button class="button" name="refresh"><i class="fa fa-refresh"></i> Refresh</button> <span style="' + Dex.getPokemonIcon('meloetta-pirouette') + ';display:inline-block;vertical-align:middle" class="picon" title="Meloetta is PS\'s mascot! The Pirouette forme is Fighting-type, and represents our battles."></span></p>';
 
 			buf += '<p><label class="label">Format:</label><button class="select formatselect" name="selectFormat">(All formats)</button></p>';
 			buf += '<label>Minimum Elo: <select name="elofilter"><option value="none">None</option><option value="1100">1100</option><option value="1300">1300</option><option value="1500">1500</option><option value="1700">1700</option><option value="1900">1900</option></select></label>';
+			buf += '<p><label>Username prefix: <input type="text" name="prefixsearch" class="textbox" value="' + BattleLog.escapeHTML(this.usernamePrefix) + '"/><button class="button search">Search</button></label></p>';
```

Comment:
I think this should match the ladder filter ('username prefix' placeholder instead of label)

Reply:
It looks out of place with no label. I'll show you what I mean in a couple minutes.

Reply:
https://i.imgur.com/vec0uat.png vs https://i.imgur.com/ykNmsJa.png.

Reply:
On second though, yours doesn't look too bad, but I feel like it's easier to read a label than reading the faded placeholder text.

Reply:
I'm just about consistently. How about we go with the placeholder for now to match and then in a followup  we can potentially change both of them to use labels?

Reply:
Sure.

Reply:
Sorry, I don't think I was clear. I didn't mean "use both", I meant, use the option from the second screenshot (no label, just placeholder) and then we can later discuss if we want to remove the placeholder in favor of labels for both filters